.eleventyignore

```diff
@@ -0,0 +1,4 @@
+.github/**/*/
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+README.md
```

Comment:
the .eleventyignore file specifies any files that exist but should _not_ be converted into HTML for the website.